containers/podman-desktop
# fix: higher-res icons for featured extensions list

### What does this PR do?

Helps address podman-desktop issue #3493 - the icons for the featured extensions are low-res which looks especially poor on high dpi displays.

I've decided based on other icon systems to set the size of the icon artwork to 512x512 px.

Note that the Developer Sandbox icon is a placeholder; I am waiting on the brand team for an update on what to use, as the logo we had been using makes usage of the now deprecated Red Hat Developer logomark. I have used the new Red Hat Developer logo as a placeholder here.

Source files for icons are available here (note this points to a private repo as it includes trademarked logo sources):
https://github.com/containers/podman-desktop-internal/commit/51376b0401c6f81cb9503b4d60d5680a3a650bc7

### Screenshot/screencast of this PR

#### Before
![Screenshot from 2023-08-11 22-44-03](https://github.com/containers/podman-desktop/assets/799683/ce677141-abed-4baf-8045-79e119b67866)

#### After
![Screenshot from 2023-08-11 23-38-06](https://github.com/containers/podman-desktop/assets/799683/0918bb93-929e-4d65-97ce-86f1cb2edd4c)

### What issues does this PR fix or reference?

[#3493](https://github.com/containers/podman-desktop/issues/3493)

### How to test this PR?

Check to make sure the icons being displayed are as appears in the "after" above and not the "before" image, and that the icons don't appear pixelated.
